participant/serializer.py

```diff
@@ -227,6 +227,9 @@ class ParticipantScoreSerializer(serializers.ModelSerializer):
     phishing_emails = serializers.SerializerMethodField()
     training_link = serializers.SerializerMethodField()
     debrief = serializers.SerializerMethodField()
+    per_email_reactions = serializers.SerializerMethodField()
+    reactions_list = serializers.SerializerMethodField()
+    reactions = []
 
     def get_phishing_emails(self, participant):
         return list(
```

Comment:
As per the updated specs we're looking to introduce 2 new fields to each phishing email.
- `participantBehaviour`
- `participantActions `

You can use the property method from `Exercise.phishing_emails` to get the list then do a simple loop to update (add) the keys above to each item in that list.

Reply:
so in context that'd be `self.exercise.phishing_emails`

Reply:
Updated specs where? I am following [221](https://github.com/cybsafe/phishtray/issues/221) (although not named them the same).

Reply:
it's 221 - check the top of the description

Reply:
the update is just the wording and what those keys contain
these extra keys still would've needed to go into each phishing email rather than separate lists